pkg/util/constants/constants.go

```diff
@@ -0,0 +1,14 @@
+package constants
```

Comment:
I don't think this needs a package - can this be declared where it is used or in the auth package?

https://github.com/golang/go/wiki/CodeReviewComments#package-names

Reply:
Some of the constants could be moved probably, but most are here because they are reused in different packages and moving them will cause an import cycle. Would it be better to declare them in every package they are used?

Reply:
Hmm - so there isn't one existing package where we can move this and re-use from other packages without causing an import cycle?

Reply:
I've tried to move it to every reasonable package but it's still causing import cycles. An alternative is to move the constants package from the `util/` dir to `authentication/`. This way at least I am not adding new packages to `util`.